control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -1547,22 +1549,42 @@ func (r *reconciler) destroyCloudResources(ctx context.Context, hcp *hyperv1.Hos
 
 	var status metav1.ConditionStatus
 	var reason, message string
+	var ltt metav1.Time
```

Comment:
what is ltt?

Reply:
LastTransitionTime

Reply:
I suspect it won't be obvious for a reader, I'd call the variable LastTransitionTime which is more obvious.